app/routes/planet_routes.py

```diff
@@ -1,10 +1,40 @@
-from flask import Blueprint, make_response, abort
-from ..models.planet import planets
+from flask import Blueprint, make_response, abort, request, Response
+from app.models.planet import Planet
+from ..db import db
 
 planets_bp = Blueprint("planets_bp",__name__,url_prefix="/planets")
 
+@planets_bp.post("")
+def create_planet():
+    request_body = request.get_json()
+    name = request_body["name"]
+    description = request_body["description"]
+    galaxy = request_body["galaxy"]
```

Comment:
We didn't explicitly look at invalid/missing data scenarios for creating a record, but think about what would happen here, and what response we might want to reply with.